model/registry/deprecated/http.yaml

```
@@ -46,6 +46,36 @@ groups:
         stability: experimental
         deprecated: "Replaced by `http.response.header.content-length`."
         examples: 3495
+      - id: client_ip
+        type: string
+        stability: experimental
+        deprecated: "Replaced by `client.address`."
+        brief: "Deprecated, use `client.address` instead."
+        examples: '83.164.160.102'
+      - id: host
+        type: string
+        stability: experimental
+        deprecated: "Replaced by one of `server.address`, `client.address` or `http.request.header.host`, depending on the usage."
+        brief: "Deprecated, use one of `server.address`, `client.address` or `http.request.header.host` instead, depending on the usage."
+        examples: ['www.example.org']
+      - id: request_content_length_uncompressed
+        stability: experimental
+        deprecated: "Removed in 1.13"
+        type: int
+        brief: "Deprecated, use `http.request.body.size` instead."
```

(just for consistency)
```suggestion
        deprecated: "Replaced by `http.request.body.size`."
        type: int
        brief: "Deprecated, use `http.request.body.size` instead."
```